python/dgl/nn/pytorch/conv/gatedgcnconv.py

```diff
@@ -0,0 +1,180 @@
+"""Torch Module for GatedGCN layer"""
+# pylint: disable= no-member, arguments-differ, invalid-name, cell-var-from-loop
+import torch
+import torch.nn.functional as F
+from torch import nn
+
+from .... import function as fn
+
+
+class GatedGCNConv(nn.Module):
+    r"""GatedGCN: Residual Gated Graph ConvNets
+        <https://arxiv.org/pdf/1711.07553v2.pdf>`
+
+    .. math::
+        e_{ij}^{l+1}=D^{l} h_{i}^{l}+E^{l} h_{j}^{l}+C^l e_{ij}^{l}
+
+        norm_{ij}=\Sigma_{j\in N_{i}} \sigma\left(e_{ij}^{l+1}\right)+\varepsilon
+
+        \hat{e}_{ij}^{l+1}=\sigma(e_{ij}^{l+1}) \div norm_{ij}
```

Comment:
Is this simply dividing `\sigma(e_{ij}^{l+1})` by `norm_{ij}`? If so, consider using `\frac{}{}` or simply `/`.